libnavigation-core/src/main/java/com/mapbox/navigation/core/replay/MapboxReplayer.kt

```diff
@@ -23,6 +23,15 @@ class MapboxReplayer {
 
     private val replayEventsObservers: MutableSet<ReplayEventsObserver> = mutableSetOf()
 
+    /**
+     * Returns whether the replay is active an playing provided events.
```

Comment:
```suggestion
     * Returns whether the replay is active and playing provided events.
```

Reply:
The "and playing provided events" is not exactly accurate. `isPlaying()` will return true under this scenario

Notice there are no events to play
``` kotlin
val mapboxReplayer = MapboxReplayer()
mapboxReplayer.play()
assertTrue(mapboxReplayer.isPlaying()
```

```suggestion
     * Returns true after calling [play] and false when the replayer is stopped
```